Updates From OlivierJM's Request

bgongre opened the pull request:
Removed flash animation on header section when hovering over images. Added a scaling animation when hovering over section titles, removed underline. Fixed syntax errors that were showing up in the console, multiple elements in the "Partners" section were using "class" instead of "className."

malgamves commented:
Sure @OlivierJM, works for me. Happy to take the lead. I added another item to the list on #102 if you're up for it @bgongre 

Also these an issue with the partner logos not being centred on mobile view. Not an emergency but worth looking to solve too

OlivierJM commented:
> Sure @OlivierJM, works for me. Happy to take the lead. I added another item to the list on #102 if you're up for it @bgongre
> 
> Also these an issue with the partner logos not being centred on mobile view. Not an emergency but worth looking to solve too

Yeah we might to focus on mobile, let's have a separate issue for that.

bgongre commented:
Commited some more changes. Updated the partners page to have a message to email "DevConZM" for partnership, added social media links in the footer, updated and cleaned up some code, and added myself to the bottom of the "all-contributorsrc" document. 👍

malgamves commented on src/components/Partners.js:
```diff
@@ -1,73 +1,86 @@
+      <div className="center email-header">
+        <h6>
+          If you would like to partner with us, send us an email at{" "}
+          <a href="mailto:info@devcon.co.zm">DevConZM</a>
```
Could you change DevConZM to the actual email address so regardless people can copy it

bgongre replied:
No worries this has been changed from the convention name to the email address so that it can be copied. It can also be clicked to open the local mail service from the users computer.

malgamves commented on src/components/Footer.js:
```diff
@@ -1,16 +1,47 @@
             {/* Copyright */}
-            <div className="container footer-copyright text-center py-3">&copy; <a href="http://devcon.co.zm">2019 DevCon Zambia.</a> All rights reserved.
+            <div className="container footer-copyright text-center py-3">
+                <p>&copy;2019 <a href="http://devcon.co.zm"> DevCon Zambia</a> all rights reserved.</p>
```
The a on all rights needs to be capitalized

bgongre replied:
Just commited and changed the "A" back to uppercase, sorry about that.

malgamves commented on src/components/Footer.js:
```diff
@@ -1,16 +1,47 @@
-import React from 'react'
+import React from 'react';
+import { FaFacebookSquare, FaTwitterSquare, FaLinkedin, FaEnvelope } from 'react-icons/fa';
```
Could you make the icons responsive and a little smaller?

bgongre replied:
In the most recent commit I have added "flex" to the social media icons to make them responsive. Also, have changed the size of the icons from 50 down to 40, let me know if they still need to be smaller.

OlivierJM commented:
@allcontributors please add @bgongre for code

OlivierJM commented:
The contribution file was edited manually, this might cause a huge conflict when merging this PR, but let's wait for all the requested changes.
@bgongre I have added you to our list of contributors and we appreciate a lot.

OlivierJM commented:
> The contribution file was edited manually, this might cause a huge conflict when merging this PR, but let's wait for all the requested changes.
> @bgongre I have added you to our list of contributors and we appreciate a lot.

I fixed the conflicts

malgamves commented:
Awesome work @bgongre

OlivierJM commented:
re-adding @bgongre on the contributors 🤦🏾‍♂️

OlivierJM commented:
@allcontrubutors please add @bgongre for code bugs

OlivierJM commented:
@allcontributors please add @bgongre for code bugs
